aboutus.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <meta name="description" content="About us of Gandhi Associates">
+    <meta name="description" content="Gandhi Associates are Investment Consultants in Shares and are dealing in Non Traded / Unlisted / Unquoted Shares.">
```

Comment:
@Shubhashree3 Update content 

**Gandhi Associates is an investment consultant in shares and are dealing in non-traded/unlisted/unquoted shares.**